static/riotjs/wizard/create_team.tag

```diff
@@ -104,13 +94,35 @@
                     window.location='/klasses/wizard/' + KLASS + '/enroll'
                 })
                 .fail(function (response) {
-                    console.log(response)
                     Object.keys(response.responseJSON).forEach(function (key) {
                         toastr.error("Error with " + key + "! " + response.responseJSON[key])
                     });
                 })
         }
 
+        self.get_eligible_leaders = () => {
```

Comment:
Only list team members in the leader dropdown.

Reply:
👍